src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    public double areaCalculator();

```

Comment:
method names should sound like action
```suggestion
    public double calculateArea();
```